frontend/src/components/Search.tsx

```diff
@@ -0,0 +1,21 @@
+import clsx from "clsx";
+import { TextField } from ".";
+
+export interface SearchProps {
+        className?: string;
+}
+
+export function Search({
+        className,
+}: SearchProps) {
+    return (
+	    <div className="p-12">
+                {/* TODO: Add search icon */}
+                <TextField label="search" placeholder="Find people and posts..." type="search" rounded={true} className={
```

Comment:
For Boolean props that are always true, you can just put the prop name (omit the `={true}` bit) and React will automatically set it to true.

E.g.
```jsx
{/* This */}
<Button disabled />
{/* Instead of this */}
<Button disabled={true} />
```